docs/runtime-contract.md

```diff
@@ -401,12 +399,12 @@ be failed.
 
 ### Default Filesystems
 
-The OCI specification describes a default container environment which may be
+The OCI specification describes a default container environment which can be
 used for many different purposes, including containerization of existing legacy
-or stateful processes which may store substantial amounts of on-disk state. In a
+or stateful processes which might store substantial amounts of on-disk state. In a
 scaled-out, stateless environment, container startup and teardown is accelerated
-when on-disk resources are kept to a minimum. Additionally, developers may not
-have access to the container filesystems (or the containers may be rapidly
+when on-disk resources are kept to a minimum. Additionally, developers might not
+have access to the container filesystems (or the containers might be rapidly
```

Comment:
I just noticed that this sentence doesn't make sense with "container filesystems". I think it was meant to say "durable storage" or "durable filesystems" (i.e. it would be fine to put all the container writes into tmpfs and throw them away when the container exits).

Reply:
'm reading this as trying to say that the developer might not be able to see the container's filesystem so they can't look at log files, etc... I think the issue might be that it should be "container's" not "container".  WDYT?

Reply:
I think the `container` -> `container's` change is sufficient here, but it could be updated further for clarity.

My understanding here is that: (1) Developers don't necessarily have access to the underlying host so you can't go looking at the stopped container contents locally. (2) Unlike docker which relies on a user to `rm` a stopped container or run `prune`, containers in this environment are likely to be cleaned up quickly after exiting.

Reply:
added the apostrophe